### ISSUE TITLE
"ReferenceError: path is not defined" in laravel-translations-loader/all.js

### ISSUE THREAD

**Issue by MrSlump** [kirschbaum-development/laravel-translations-loader#13]
I got built error through invoking storybook.

```
ERROR in ./node_modules/@kirschbaum-development/laravel-translations-loader/all.js (./node_modules/@kirschbaum-development/laravel-translations-loader/all.js!./node_modules/@kirschbaum-development/laravel-translations-loader/all.js)
Module build failed (from ./node_modules/@kirschbaum-development/laravel-translations-loader/all.js):
ReferenceError: path is not defined
```

I have to workaround like this:
```
laravel-translations-loader (master) $ git diff all.js
diff --git a/all.js b/all.js
index 7710b5c..16d8ced 100644
--- a/all.js
+++ b/all.js
@@ -1,4 +1,5 @@
 const _ = require('lodash');
+const path = require('path');
 const phpLoader = require('./php-loader');
```

Could you fix it?

#### info
```
Node: v10.9.0
@storybook/vue: v5.0.10
webpack: 4.30.0
```

**Comment by luisdalmolin** [comment]
@MrSlump I'll check this out next week, this week I'm fully out. If you downgrade to `0.4.*` version in the meantime should be good.

**Comment by adnedelcu** [comment]
I created a pull request that fixes this issue along with another one where you can't use sometimes both `parameters` and `namespace` options.

**Comment by MrSlump** [comment]
@luisdalmolin @adnedelcu

Thanks for helping me!
#14 is well-running in my storybook.

**Comment by luisdalmolin** [comment]
Just FYI that the PR #14 was merged and tagged as 0.6.0.
